src/commcare_cloud/ansible/roles/kafka/files/log4j_new.properties

```diff
@@ -0,0 +1,74 @@
+log4j.rootLogger=INFO, stdout, kafkaAppender
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+log4j.appender.kafkaAppender=org.apache.log4j.DailyRollingFileAppender
+log4j.appender.kafkaAppender.DatePattern='.'yyyy-MM-dd-HH
+log4j.appender.kafkaAppender.File=${kafka.logs.dir}/server.log
```

Comment:
are these meant to be resolved by ansible? If so it needs to be moved to the `templates` folder and the syntax is `{{var_name}}`

Reply:
This variable is of kafka